libr/asm/arch/gb/gbasm.c

```diff
@@ -159,6 +166,38 @@ static bool gb_parse_ld2 (ut8 *buf, char *buf_asm) {
 	return false;
 }
 
+static bool gb_parse_ld3 (ut8 *buf, char *buf_asm) {
+	if (strlen (buf_asm) < 7) {
+		return false;
+	}
+	if (buf_asm[5] != ',') {
+		return false;
+	}
+
+	const ut32 reg = (buf_asm[3] << 8) | buf_asm[4];
+	switch (reg) {
+	case 0x6263:	//bc
+		buf[0] = 0x01;
+		break;
+	case 0x6465:	//de
+		buf[0] = 0x11;
+		break;
+	case 0x686c:	//hl
+		buf[0] = 0x21;
+		break;
+	case 0x7370:	//sp
+		buf[0] = 0x31;
+		break;
+	default:
+		return false;
+	}
+
+	const ut64 num = r_num_get (NULL, &buf_asm[6]);
```

Comment:
use buf_asm + 6, its easier to read and shorter

Reply:
it is shorter, but harder to read